CCatHome_Regional_Host_Guide.md

```diff
@@ -0,0 +1,228 @@
+
+- *If the Session Host is not present (e.g. unable to connect or attend), you can take the host role (follow the instructions listed for the host). If you are uncomfortable being the host, please immediately contact the taskforce so that one of them can come to this session to support you.*
+
+- Observe the chat to make sure that there are no inappropriate posts or comments.
+
+- Keep an eye on your personal/direct message during the session in case someone needs to report something during the call
+
+- Keep the [Code of conduct shared folder](https://drive.google.com/drive/folders/1iQX5TFWuXmeXERgqR4zI0QaAulEj_6OD?usp=sharing) and [CoC Facilitators Roles and Responsibilities](https://docs.google.com/document/d/1p0GIrNzXHYJlJmFr_18oOkOIFRnfH51BqlBmqhNni-Y/edit?usp=sharing) in handy in case you have to react to a report by relaying them to the CoC committee
+
+- Help the Session Facilitator manage participants by e.g. muting those who aren't speaking. (Open the "Participants" tab → find their name → click "Mute")
+
+#### At the end of the session
+
+- Press the stop button for recording.
+- Make sure you send a direct email to the CarpentryCon @ home taskforce to let them know if there was an incident reported in your session.
+- If there was no incident to report, please send an email stating "There was no incident reported at the session \<name of the session\>.
+
+### Instructions for the session leads (speakers/chairs/facilitators)
+
+#### General instructions
+
+- Types of sessions at CarpentryCon @ home: [https://github.com/carpentrycon/carpentryconhome-proposals/blob/master/README.md](https://github.com/carpentrycon/carpentryconhome-proposals/blob/master/README.md)
+- Speakers guide
+- talks
+- workshop
+- collaborative sessions
+- etc
+
+#### Before the sessions
+
+- Your regional host should contact with you the connection details for your session
+- Please check the agenda and add your details in your calendar
+- Please let the CarpentryCon @ home taskforce and your regional host know if any changes in your session have been made after the approval of your proposal, and if at all you don't want to record your session
+- Please read the [Zoom Instructions](https://docs.google.com/document/d/1_7vxMtcqdn8s-zYOG6THnxZvs4uYmfJUySjNVeVFj98/edit?usp=sharing) and test you Zoom account
+- Connect with your session helpers to plan your session
+- If possible, please put materials for your session on your personal or the CarpentryCon @ Home proposal repository (by a pull request): [https://github.com/carpentrycon/carpentryconhome-proposals](https://github.com/carpentrycon/carpentryconhome-proposals)
+- If possible, publish your slides and other materials on Zenodo, FigShare or other open platforms.
+- Add more details and customise the shared nodes / collaborative documents prepared by the task force. Make sure the document contains links to all relevant resources.
+- Link details for your session the shared notes shared by your regional host
```

Comment:
I'm not sure what this sentence means?

Reply:
Me neither 🤣
I'm removing this.

Reply:
Some suggestions mentioned inside the asterisks:

- Add more details and customize the shared *notes* / collaborative documents prepared by the task force. Make sure the document contains links to all relevant resources.
- *Link any additional details for your session in the shared notes (you should have received an Etherpad link from your regional host)*

Reply:
thank you @malvikasharan I've added it now